run_pipeline.sh

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
 
-./packages/cli/bin/index.js run ./test/pipelines/$1.json
+chmod +x ./packages/cli/dist/bin/index.js
```

Comment:
This could be finished offline.

Reply:
This needs to be resolved.

Reply:
if not chmod, it will throw error when run pipline workflow because "./packages/cli/dist/bin/index.js" has not an executable rights

Reply:
because file "./packages/cli/dist/bin/index.js" is generated after build but not write in 'bin' dir like before:(

Reply:
See https://github.com/microsoft/TypeScript/issues/37583, this is still requesting and awaiting the feedback from TypeScript team, but we still could use ts-node to address this issue, could you please take a look at https://www.geekytidbits.com/cli-apps-in-typescript-via-ts-node/?

Reply:
Resolved, could you please take a look:(